pkg/syncer/metadatasyncer.go

```diff
@@ -2640,3 +2655,139 @@ func initResizeReconciler(ctx context.Context, tkgClient clientset.Interface,
 	rc.Run(ctx, resizeWorkers)
 	return nil
 }
+
+// createStoragePolicyUsageCR creates StoragePolicyUsage CR with given parameters
+func createStoragePolicyUsageCR(ctx context.Context, quotaClient client.Client, name, namespace,
+	storagePolicyId, storageClassName, resourceKind, resourceApiGroup,
+	extensionName string) (*storagepolicyusagev1alpha1.StoragePolicyUsage, error) {
+	newUsageInstance := storagepolicyusagev1alpha1.StoragePolicyUsage{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       cnsoperatorv1alpha1.GroupName,
+			APIVersion: cnsoperatorv1alpha1.Version,
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:              name,
+			Namespace:         namespace,
+			Generation:        0,
+			CreationTimestamp: metav1.Time{},
+		},
+		Spec: storagepolicyusagev1alpha1.StoragePolicyUsageSpec{
+			StoragePolicyId:       storagePolicyId,
+			StorageClassName:      storageClassName,
+			ResourceKind:          resourceKind,
+			ResourceAPIgroup:      &resourceApiGroup,
+			ResourceExtensionName: extensionName,
+		},
+	}
+	err := quotaClient.Create(ctx, &newUsageInstance, &client.CreateOptions{})
+	if err != nil {
+		log.Errorf("Failed to create StoragePolicyUsage for policyID %v storageclass %v resourceKind %v. Err: %+v",
```

Comment:
This is using "github.com/aunum/log"? Can we use the same logging mechanism thru "ctx, log := logger.GetNewContextWithLogger()"?

Reply:
Used log.GetLogger() at the top of the method